src/envo/envo-edit.owl

```diff
@@ -37862,6 +38067,1591 @@ AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibra
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_01002127> "2024-12-16T17:26:18Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_01002127> "mangrove abundance"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01002127> <http://purl.obolibrary.org/obo/ENVO_01002119>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_01002128> (microbial abundance)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01002128> "Ecological abundance which inheres in a collection of microorganisms.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_01002128> <https://orcid.org/0000-0002-4366-3088>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_01002128> "2024-12-16T17:26:18Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_01002128> "microbial abundance"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01002128> <http://purl.obolibrary.org/obo/ENVO_01002119>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_01002129> (phytoplankton abundance)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01002129> "Ecological abundance which inheres in a collection of phytoplankters.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_01002129> <https://orcid.org/0000-0002-4366-3088>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_01002129> "2024-12-16T17:26:18Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_01002129> "phytoplankton abundance"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01002129> <http://purl.obolibrary.org/obo/ENVO_01002119>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_01002130> (seagrass abundance)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01002130> "Ecological abundance which inheres in a collection of seagrass plants.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_01002130> <https://orcid.org/0000-0002-4366-3088>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_01002130> "2024-12-16T17:26:18Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_01002130> "seagrass abundance"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01002130> <http://purl.obolibrary.org/obo/ENVO_01002119>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_01002131> (turtle abundance)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01002131> "Ecological abundance which inheres in a collection of organisms from the Order Testudines.")
```

Comment:
We dont say "inheres in" anymore nowadays, when say "characteristic of" and "has characteristic" (https://www.ebi.ac.uk/ols4/ontologies/ro/properties/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FRO_0000052)